vignettes/delta-method.Rmd

```diff
@@ -176,15 +176,15 @@ print(mean_pred_treat)
 
 ## Variance
 
-So far this is pretty straightforward, but we want to know the standard error of these estimates. As explained above, we can approximate their variance: if the gradient $\nabla {\hat{y}_{\tau = t}}$ is defined as $\mathbf{J}$, then
+So far this is pretty straightforward, but we want to know the standard error of these estimates. As explained above, we can approximate their variance: if the gradient $\nabla g(\hat{y})$, the derivative with respect to each $\beta$ parameter, is defined as $J$, then
 
-\[\text{Var}(\hat{y}_{\tau = t}) =  \mathbf{J} \times \mathbf{V} \times \mathbf{J}^\prime\]
+\[\text{Var}(g(\hat{y})) =  J \times V \times J^\prime\]
 
-\ldots and $\nabla {\hat{y}_{\tau = t}}$ is the derivative of $\hat{y}_{\tau = t}$ with respect to each of the $\beta$ parameters. Going back to our function from before, let's start with the coefficient on treatment, $\beta_1$.
+\ldots Going back to our function from before, let's start with the coefficient on treatment, $\beta_1$.
```

Comment:
Tried to shorten this whole section for consistency / accuracy / removing redundancy.